### DISCUSSION TITLE
Manually Remember Databases

### DISCUSSION THREAD

**Discussion by trymeouteh** [Kunzisoft/KeePassDX#1739] [Ideas]
Have a setting to "save" a database as a remember database.

This way you can have the "remember database locations" setting disabled, but manually add remember databases to keepassdx.

This is useful for when you want to have a few databases remembered but when you open other databases, to not have them remembered.

**Comment by J-Jamet** [comment]
Issue : https://github.com/Kunzisoft/KeePassDX/issues/1748
